pkg/api/v1beta1/conversion.go

```diff
@@ -444,7 +444,115 @@ func init() {
 			}
 			return nil
 		},
+		// Converts cpu and memory requirements to Resources, and vice versa.
+		func(in *newer.Container, out *Container, s conversion.Scope) error {
+			if err := s.Convert(&in.Name, &out.Name, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.Image, &out.Image, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.Command, &out.Command, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.WorkingDir, &out.WorkingDir, 0); err != nil {
+				return err
+			}
 
+			if err := s.Convert(&in.Ports, &out.Ports, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.Env, &out.Env, 0); err != nil {
+				return err
+			}
+			if err := s.Convert(&in.VolumeMounts, &out.VolumeMounts, 0); err != nil {
```

Comment:
Please put these in exactly the same order as in types.go. Otherwise, it's hard to quickly determine that all fields are covered.